mass_sorting/views/mass_sort_wizard_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2016-Today GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<openerp><data>
+
+    <record id="view_mass_sort_wizard_form" model="ir.ui.view">
+        <field name="model">mass.sort.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Mass Sorting" version="7.0">
+                <group col="4">
+                    <field name="description" nolabel="1" colspan="4" attrs="{'invisible': [('allow_custom_setting', '=', True)]}"/>
+                    <field name="line_ids" colspan="4" nolabel="1" attrs="{'invisible': [('allow_custom_setting', '=', False)]}">
+                        <tree editable="bottom">
+                            <field name="sequence" widget="handle"/>
+                            <field name="field_id"/>
```

Comment:
Good idea, but there is a bug in Odoo core. When I set such widget, Odoo loose domain. So I didn't commited your suggestion.